src/routes/applications.ts

```diff
@@ -350,6 +350,32 @@ const createApplicationsRouter = (
     }),
   );
 
+  router.post(
+    '/applications/:id/renew',
+    authFilter([]),
+    wrapAsync(async (req: Request, res: Response, next: NextFunction) => {
+      const id = req.params.id;
+      const validatedId = validateId(id);
+      const identity = (req as IRequest).identity;
```

Comment:
We should look into using a system that communicates to Typescript automatically that the Request implements the `IRequest` interface instead of just assuming it and coercing it.